index.js

```diff
@@ -4030,6 +4030,25 @@
   S.splitOn =
   def('splitOn', {}, [$.String, $.String, $.Array($.String)], splitOn);
 
+  //# sort :: ((a, a) -> Number) -> Array a -> Array a
+  //.
+  //. Returns a copy of the list, sorted according to the comparator
+  //. function, which should accept two values at a time and return a
+  //. negative number if the first value is smaller, a positive number if
+  //. it's larger, and zero if they are equal. Please note that this is a
+  //. **copy** of the list. It does not modify the original.
+  //.
+  //. ```javascript
+  //. > S.sort(S.sub, [4, 2, 7, 5])
+  //. [2, 4, 5, 7]
+  //. ```
+  function sort(f, a) {
+    return a.sort(f);
```

Comment:
I have a comment here: https://github.com/sanctuary-js/sanctuary/issues/353#issuecomment-288889085


So, what about `def('sortBy', {f: [Z.Applicative, Z.Foldable, Z.Monoid]}, [Fn(a, Fn(a, Ordering)), f(a), f(a)], sortBy);` ?

Then we could use something like this: `sortBy cmp = foldr (insertBy cmp) []`, or should we rather implement the better/harder merge sort that's over here? http://hackage.haskell.org/package/base-4.9.1.0/docs/src/Data.OldList.html#sortBy

Little question: Do we have foldr in sanctuary-js?